### ISSUE TITLE
Not working on nested member expression

### ISSUE THREAD

**Issue by cpylua** [remithomas/eslint-plugin-ban#8]
## Config

```
"ban/ban": [
  "error",
  {
    "name": ["*", "removeEventListener"],
    "message": "Use xx instead"
  }
]
```

## Expected

`this.target.removeEventListener` should be an error.

## Current behavior

No error reported on `this.target.removeEventListener`

**Comment by remithomas** [comment]
Hey @cpylua, thanks for your report.
Indeed it should throw an error, I will fix that as soon as possible.

**Comment by remithomas** [comment]
New version 1.4.0 published.
Thanks again for the report @cpylua 👍
